shared_infra/dynamo_write_heartbeat/src/dynamo_write_heartbeat.py

```diff
@@ -0,0 +1,20 @@
+# -*- encoding: utf-8 -*-
+"""
+Run a no-op write on DynamoDB (delete a non existent item) to be run in a scheduled lambda to
+make DynamoDB scale down when there would otherwise be zero throughput and it otherwise wouldn't.
+
+TABLE_NAMES: comma separated list of dynamoDb tables to call
+"""
+
+import boto3
+import os
+
+from wellcome_aws_utils.lambda_utils import log_on_error
+
+@log_on_error
+def main(event, context, endpoint_url=None):
+    dynamodb = boto3.resource('dynamodb', endpoint_url=endpoint_url)
+    table_names = os.environ.get('TABLE_NAMES').split(",")
```

Comment:
This will mask an error if the environment variable isn't set:

```
AttributeError: 'NoneType' object has no attribute 'split'
```

A possible alternative:

```python
try:
    table_names = [t.strip() for t in os.environ['TABLE_NAMES']]
except KeyError:
    raise RuntimeError(f'TABLE_NAMES not found in os.environ={os.environ!r}')
```